src/ucp/core/ucp_context.c

```diff
@@ -195,6 +199,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "RNDV fragment size \n",
    ucs_offsetof(ucp_config_t, ctx.rndv_frag_size), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"MEM_TYPE_PTR_CACHE", "y",
+   "Enable memory type(cuda) pointer cache \n",
```

Comment:
need UCP-level unit tests for 2 modes: with and without cache (at least some basic tests which trigger memotype check)